mmeds/html/analysis_select_tool.html

```diff
@@ -12,7 +12,7 @@
         <label> Select Analysis Type </label>
```

Comment:
Q: if we end up having a bunch of tools, wouldn't we want to populate the lines below from some data structure that contains those?

Reply:
Yes, that wouldn't be hard at all, there's already html and python code for doing that elsewhere in MMEDS. Currently it would just have the one tool though